Xamarin.Forms.Core/Shell/Shell.cs

```diff
@@ -402,32 +402,31 @@ internal static void ApplyQueryAttributes(Element element, IDictionary<string, s
 				prefix = route + ".";
 			}
 
-			var type = element.GetType();
-			var typeInfo = type.GetTypeInfo();
-#if NETSTANDARD1_0
-			var effectAttributes = typeInfo.GetCustomAttributes(typeof(QueryPropertyAttribute), true).ToArray();
-#else
-			var effectAttributes = typeInfo.GetCustomAttributes(typeof(QueryPropertyAttribute), true);
-#endif
-
-			if (effectAttributes.Length == 0)
-				return;
+			//if the lastItem is implicitly wrapped, get the actual ShellContent
+			if (isLastItem) {
+				if (element is ShellItem shellitem && shellitem.Items.FirstOrDefault() is ShellSection section)
```

Comment:
"Implicitly wrapped" implies to me a single element in the list. So, instead of `FirstOrDefault` I'd expect `SingleOrDefault`. Of course, I also live in grass shack and commute in a canoe, so...